metrics/report/report_dockerfile/collectd_scaling.R

```diff
@@ -314,7 +318,27 @@ for (currentdir in resultdirs) {
 				}
 				min_idle_cpu=node_cpu_idle_data$value[cpu_end_index]
 
-				cputotal = cputotal + (max_idle_cpu - min_idle_cpu)
+				# Use a linear regression model to try and guesstimate the CPU
```

Comment:
I was going to note - if there is one bit of code that needs somebody else to really stare at it - it is this `lm()` usage - anybody out there got some good background in statistics?...

Reply:
Wow, this is interesting... :)
I'll need to try with few test rounds to see real values that linear regression gives from the data I have.
In manual debugging I've used averages over few measurements at the start and end of the collectd cpu data, so this one is pretty advanced. :)

Reply:
:-) Yeah, I wanted to work out how to 'fit a curve/line to the data I could then take measures from'. I'm definitely open to other suggestions as well.

Reply:
A note in passing - there may be another way to extract the interpolated/intersection points using the R [predict](https://www.rdocumentation.org/packages/raster/versions/3.0-7/topics/predict) function. The following article is worth a read:

https://aosmith.rbind.io/2018/11/16/plot-fitted-lines/